src/definitions.js

```diff
@@ -0,0 +1,16 @@
+import { Record } from 'immutable';
+
+export function definitionToRecord(recordName, definition) {
+  // Create object with keys of definition and 'null' values. All object types have a
+  // 'metadata' key.
+  let record = {'metadata': null};
+
+  for (const prop in definition) {
+    if (definition.hasOwnProperty(prop)) {
+      record[prop] = null;
+    }
+  }
```

Comment:
Is dit niet dubbel-op-dubbel-op? wanneer "prop in definition", dan zit deze key dus in het object en is hasOwnProperty altijd waar?